modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/CorrelatedSubqueryPlannerTest.java

```diff
@@ -68,7 +68,7 @@ public void test() throws Exception {
                 + "    OR c>d\n"
                 + " ORDER BY 1;";
 
-        IgniteRel rel = physicalPlan(sql, schema, "FilterTableScanMergeRule");
+        IgniteRel rel = physicalPlan(sql, schema, "FilterTableScanMergeRule", "IgniteProjectCorrelateTransposeRule");
```

Comment:
why did you disable new rule in this test?

Reply:
This requires some reworking of the test, but I don't see the need for it.

Reply:
test has been reworked